### PR TITLE
Add Verified Id Style to VCVerifiedId and improve test app

### PR DESCRIPTION
**Pull request by symorton**
- Add BasicVerifiedIdStyle to VCVerifiedId and improve test app

### CODE REVIEW

File: Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift [hunk 1]
```diff
@@ -8,20 +8,54 @@ import WalletLibrary
...
     
-    mutating func save(verifiedId: VerifiedId) throws {
-        storedVerifiedIds.append(verifiedId)
+    let store = PersistenceController.shared.container
+    
+    init(verifiedIdClient: VerifiedIdClient) {
+        self.verifiedIdClient = verifiedIdClient
+    }
+    
+    func save(verifiedId: VerifiedId) throws {
```
Comment (nithyaganeshng) [inline Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift:19]: Is this saving in database?

File: Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift [hunk 2]
```diff
@@ -8,20 +8,54 @@ import WalletLibrary
...
+        newVerifiedId.raw = rawVerifiedId
+        try viewContext.save()
     }
     
-    mutating func delete(verifiedId: VerifiedId) throws {
-        storedVerifiedIds.removeAll {
-            $0.id == verifiedId.id
+    func delete(verifiedId: VerifiedId) throws {
+        let viewContext = store.viewContext
+        let request = NSFetchRequest<NSFetchRequestResult>(entityName: "RawVerifiedId")
```
Comment (nithyaganeshng) [inline Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift:29]: nit: may be name it as rawVerifiedId? It is confusing with request object.

File: WalletLibrary/WalletLibrary/Extensions/Entities/IssuanceResponseContainer+WalletLibrary.swift [hunk 3]
```diff
@@ -59,14 +59,14 @@ extension IssuanceResponseContainer: IssuanceResponseContaining {
     }
     
     private mutating func add(groupRequirement: GroupRequirement) throws {
-        try groupRequirement.validate()
+        try groupRequirement.validate().get()
```
Comment (nithyaganeshng) [inline WalletLibrary/WalletLibrary/Extensions/Entities/IssuanceResponseContainer+WalletLibrary.swift:62]: what does this get() do?

**Review by nithyaganeshng — APPROVED** [review]

File: WalletLibrary/WalletLibrary/Extensions/Entities/IssuanceResponseContainer+WalletLibrary.swift (hunk 3 above)
Comment (symorton) [reply WalletLibrary/WalletLibrary/Extensions/Entities/IssuanceResponseContainer+WalletLibrary.swift:62]: it just turns a Result into the object or throws.

File: Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift (hunk 2 above)
Comment (symorton) [reply Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift:29]: it is actually a database request.

File: Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift (hunk 1 above)
Comment (symorton) [reply Demo/WalletLibraryDemo/WalletLibraryDemo/VerifiedIdRepository.swift:19]: yes

**Pull request merged**